### PR TITLE
feat: accept we travel together

### PR DESCRIPTION
**Pull request by hanggarHungry24**
ticket basecamp: https://3.basecamp.com/5190892/buckets/31126446/todos/5916411103

### CODE REVIEW

File: partial/booking/BookingAcceptWeTravelTogether.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      :phone-code="'+66'"
+      v-model:phone="weTravelTogether.wttPhone"
+      v-model:identityCardNumber="weTravelTogether.wttIdCard"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineAsyncComponent } from "vue";
+import ActivateButton from "@/section/booking/BookingAcceptWeTravelTogether/ActivateButton.vue";
```
Comment (afif-hh) [inline partial/booking/BookingAcceptWeTravelTogether.vue:24]: nama folder nya pakai huruf kecil sama `_` mas

File: partial/booking/BookingAcceptWeTravelTogether.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { watch } from "vue";
+import { storeToRefs } from "pinia";
+import { useBookingStore, useBookingChargeStore } from "@/stores/booking";
```
Comment (afif-hh) [inline partial/booking/BookingAcceptWeTravelTogether.vue:27]: @ diganti ke ~

File: partial/booking/BookingAcceptWeTravelTogether.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { useUserVoucherStore, useVoucherStore } from "~~/stores/voucher";
```
Comment (afif-hh) [inline partial/booking/BookingAcceptWeTravelTogether.vue:28]: remove double `~`

File: partial/booking/BookingAcceptWeTravelTogether.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+const voucherStore = useVoucherStore();
+const userVoucherStore = useUserVoucherStore();
+const FormBooking = defineAsyncComponent(
+  () =>
+    import("@/section/booking/BookingAcceptWeTravelTogether/FormBooking.vue")
+);
+const bookingStore = useBookingStore();
+const { isAcceptWeTravelTogetherActive, weTravelTogether } =
```
Comment (afif-hh) [inline partial/booking/BookingAcceptWeTravelTogether.vue:37]: kyk nya enak `isAcceptWeTravelTogetherActive` diganti ke `isAcceptWeTravelTogether` aja dan `weTravelTogether` ke `weTravelTogetherForm` mas

File: section/booking/BookingAcceptWeTravelTogether/ActivateButton.vue [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="flex justify-between">
+    <div class="flex items-center gap-1">
+      <img
+        src="@/assets/image/icon-accept-travel-together.png"
```
Comment (afif-hh) [inline section/booking/BookingAcceptWeTravelTogether/ActivateButton.vue:5]: pakai ~ aja mas

File: stores/booking.ts [hunk 6]
```diff
@@ -89,9 +89,27 @@ const useBookingStore = defineStore("booking", {
...
+      isAcceptWeTravelTogetherActive: false,
+      weTravelTogether: {
+        wttIdCard: "",
+        wttPhone: "",
+      },
     };
   },
-  getters: {},
+  getters: {
+    isAcceptWeTravelTogetherDisabled(state) {
```
Comment (afif-hh) [inline stores/booking.ts:100]: diganti ke `isValidWetravelTogether` aja mas

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: partial/booking/BookingAcceptWeTravelTogether.vue (hunk 1 above)
Comment (hanggarHungry24) [reply partial/booking/BookingAcceptWeTravelTogether.vue]: updated mas

File: partial/booking/BookingAcceptWeTravelTogether.vue (hunk 3 above)
Comment (hanggarHungry24) [reply partial/booking/BookingAcceptWeTravelTogether.vue]: updated mas

File: section/booking/BookingAcceptWeTravelTogether/ActivateButton.vue (hunk 5 above)
Comment (hanggarHungry24) [reply section/booking/BookingAcceptWeTravelTogether/ActivateButton.vue]: updated mas

File: stores/booking.ts (hunk 6 above)
Comment (hanggarHungry24) [reply stores/booking.ts]: updated mas

**Review by afif-hh — APPROVED** [review]

**Pull request merged**